src/main/java/frc/team3128/commands/CmdAlign.java

```diff
@@ -62,6 +65,7 @@ public void execute() {
                     currHorizontalOffset = limelights.getShooterTX();
                     prevError = goalHorizontalOffset - currHorizontalOffset;
                     aimState = VisionState.FEEDBACK;
+                    controller.disableContinuousInput();
```

Comment:
this should be .enable not .disable because you're switching from searching to feedback